policy/modules/contrib/slocate.fc

```diff
@@ -3,5 +3,6 @@
 /usr/bin/updatedb.*	--	gen_context(system_u:object_r:locate_exec_t,s0)
 
 /var/lib/[sm]locate(/.*)?	gen_context(system_u:object_r:locate_var_lib_t,s0)
+/var/lib/plocate(/.*)?		gen_context(system_u:object_r:locate_var_lib_t,s0)
```

Comment:
Or maybe do `/var/lib/[smp]locate(/.*)?	gen_context(system_u:object_r:locate_var_lib_t,s0)` instead? That'd be more consistent.

Reply:
I don't understand. If line 5 is ok, then just adding one more character there shouldn't change anything.

Reply:
On selinux-policy updates, rpm script identifies changed files and runs fixfiles for them. If any filename contains a regexp, it is converted to "*" without further distinguishing and passed to restorecon.

https://github.com/SELinuxProject/selinux/blob/master/policycoreutils/scripts/fixfiles#L170

Reply:
Wow.

Reply:
I dare to read that as approved.

Reply:
Correct :)